__mocks__/swrv.js

```diff
@@ -2,6 +2,9 @@ const baseURL = import.meta.env.VITE_TRACTION_BASE_URL
 
 const flags = {
   flipper_id: 'User',
+  features: {
```

Comment:
Where are these used?
Maybe worth adding some documentation here because I'm not sure what is accessing the feature flags through this.

Reply:
You are right. It is not required, it seems which I thought is used  while mocking